sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/HttpResponseHeaderDecoder.java

```diff
@@ -75,7 +75,7 @@
      * @param headers the REST API returned headers
      * @return instance of header entity type created based on provided {@headers}, if header entity model does
      * not exists then return null
-     * @throws IOException
+     * @throws IOException If an I/O error occurs
```

Comment:
Should we do a checkstyle ignore on these implementation classes?

Reply:
@JonathanGiles

Reply:
Yes. We only care about public API. Things inside the implementation package are not public API and don't need to be checked for JavaDoc validity.

Reply:
Done.